e2e/oidc-suites/src/user.spec.ts

```diff
@@ -27,11 +27,11 @@ test.describe('User tests', () => {
     await page.getByRole('textbox', { name: 'Password' }).fill(pingAmPassword);
     await page.getByRole('button', { name: 'Next' }).click();
 
-    await page.waitForURL('http://localhost:8443/ping-am/**');
+    await page.waitForURL('http://localhost:8443/ping-am/**', { waitUntil: 'networkidle' });
```

Comment:
I think `networkidle` is an anti-pattern or discouraged.

Reply:
```
'networkidle' - DISCOURAGED consider operation to be finished when there are no network connections for at least 500 ms. Don't use this method for testing, rely on web assertions to assess readiness instead.
```

Reply:
This true but we use it a lot in our tests rights now. We have a ticket to do a bunch of Playwright updates later on.
https://pingidentity.atlassian.net/browse/SDKS-4331

Reply:
Updated the ticket